helmfile/values/service-cluster-networkpolicy.yaml.gotmpl

```diff
@@ -18,7 +18,6 @@ global:
     ports: {{- toYaml .Values.networkPolicies.global.objectStorage.ports | nindent 6 }}
   externalLoadBalancer: {{ .Values.networkPolicies.global.externalLoadBalancer }}
   ingressUsingHostNetwork: {{ .Values.ingressNginx.controller.useHostPort }}
-
 harbor:
```

Comment:
Why is this removed? We seem to have a new line between the other components.

Reply:
Unintended.